feature/profile/src/commonTest/kotlin/io/github/droidkaigi/confsched/profile/ProfileScreenTest.kt

```diff
@@ -0,0 +1,144 @@
+import io.github.droidkaigi.confsched.testing.robot.profile.ProfileDataStoreRobot.ProfileInputStatus
+import io.github.droidkaigi.confsched.testing.robot.profile.ProfileScreenRobot
+
+@RunWith(UiTestRunner::class)
+class ProfileScreenTest {
+    val testAppGraph = createProfileScreenTestGraph()
+
+    @ComposeTest
+    fun runTest() {
+        describedBehaviors.forEach { behavior ->
+            val robot = testAppGraph.profileScreenRobotProvider()
+            runComposeUiTest {
+                behavior.execute(robot)
+            }
+        }
+    }
+
+    val describedBehaviors = describeBehaviors<ProfileScreenRobot>("ProfileScreen") {
+        describe("when profile card is does not exists") {
+            doIt {
+                setupProfileDataStore(ProfileInputStatus.AllNotEntered)
+                setupProfileScreenContent()
+            }
+            itShould("show profile edit screen") {
+                captureScreenWithChecks {
+                    checkEditScreenDisplayed()
+                }
+            }
+            describe("fill out all text forms") {
+                val profile = Profile.fake()
+                doIt {
+                    inputName(profile.nickName)
+                    inputOccupation(profile.occupation)
+                    inputLink(profile.link)
+                }
+                itShould("show the entered information correctly") {
+                    captureScreenWithChecks {
+                        checkName(profile.nickName)
+                        checkOccupation(profile.occupation)
+                        checkLink(profile.link)
+                    }
+                }
+            }
+            describe("when changing card themes") {
+                doIt {
+                    clickCardTheme(ProfileCardTheme.LightDiamond)
+                }
+                itShould("is changed to the selected theme") {
+                    captureScreenWithChecks {
+                        checkThemeSelected(ProfileCardTheme.LightDiamond)
+                    }
+                }
+            }
+            describe("click create card button when profile is empty") {
+                doIt {
+                    clickCreateButton()
+                }
+                itShould("show error text") {
+                    captureScreenWithChecks {
+                        checkNameError()
+                        checkOccupationError()
+                        checkLinkEmptyError()
+                        checkImageEmptyError()
+                    }
+                }
+            }
+            describe("check invalid link") {
+                doIt {
+                    inputLink("あいうえお")
+                    clickCreateButton()
+                }
+                itShould("show error text") {
+                    captureScreenWithChecks {
+                        checkLinkInvalidError()
+                    }
+                }
+            }
```

Comment:
Could you check if these screenshot results are as intended?
You might need to scroll down to include the part with the error message.

<img width="796" height="648" alt="image" src="https://github.com/user-attachments/assets/74c1e575-5d0b-4fd0-93cc-d58825bc76d8" />

Reply:
It might be hidden under the TopAppBar. I recall a similar workaround being applied in the AboutScreen screenshot test as well.

https://github.com/DroidKaigi/conference-app-2025/blob/9015f6b2e612e57768733a6635cf7d2edc22a71a/core/testing/src/commonMain/kotlin/io/github/droidkaigi/confsched/testing/robot/about/AboutScreenRobot.kt#L78-L81